e.js

```diff
@@ -196,6 +196,19 @@ var inject = function () {
                             a.textContent = "Copy ID";
                         }, 1000)
                     });
+                    /* add Open task in another tab */
+                    const tags = $(".oe_kanban_content", this);
+                    const b = document.createElement("b");
+                    b.setAttribute("role", "menuitem");
+                    b.setAttribute("modifiers", "#");
+                    b.setAttribute("href", "#");
+                    b.classList.add("dropdown-item", "oe_kanban_action", "oe_kanban_action_a");
+                    tags.append(b);
+                    b.textContent = "Open Task";
+                    b.addEventListener("click", () => {
+                        const url = `https://www.odoo.com/web#id=${id}&cids=1&menu_id=4720&action=333&active_id=49&model=project.task&view_type=form`;
+                        window.open(url, '_blank').focus();
+                    });
```

Comment:
```suggestion
                    const b = document.createElement("b");
                    b.innerHTML = `<a href="https://www.odoo.com/web#id=${id}&cids=1&menu_id=4720&action=333&active_id=49&model=project.task&view_type=form" target="_blank">Open Task</a>`;
                    tags.append(b);
```
and style wise,  it would be better to have it one same line as "Copy ID" (maybe shorter string, or having awesome font)